NavigateU/NavigateU/HomeScreen/Views/HomeScreenView.swift

```diff
@@ -7,21 +7,31 @@
 
 import UIKit
 
-class HomeScreenView: UIView {
+protocol HomeScreenDelegate: AnyObject {
```

Comment:
there is the best way to call protocol - is add "delegate" keyword to full name of your entity - HomeScreenViewDelegate